### PR TITLE
Fix linking on Mac

### PR DESCRIPTION
**Pull request by Nadrieril**
I accidentally set a linux-only toolchain in #133. This should fix `make test` on Mac

### CODE REVIEW

**Review by sonmarcho — COMMENTED** [review]
Looks good to me. @R1kM I'll let you approve this and merge it if it works on your side.

**Comment by msprotz** [comment]
Works for me

**Review by msprotz — APPROVED** [review]

**Pull request merged**
